datagrepper/util.py

```diff
@@ -69,16 +66,24 @@ def assemble_timerange(start, end, delta):
     if delta is not None:
         delta = float(delta)
 
+    default_delta = float(default_delta)
```

Comment:
Why not use `app.config["DEFAULT_QUERY_DELTA"]` here? (with `flask.current_app`). Then you don't need an additional argument.

Reply:
isn't it simpler to not require the utils to be aware of the flask application here?

is there a negaive to passing an additional parameter through here?

Reply:
i tried and failed to use current_app, running into issues wiht the flask application context, so i also tried to fix that and failed.

added it as an extra commit if you have suggestions, but TBH, would like to know if this is worth it to just avoid passing an extra parameter to the util function.

Reply:
> isn't it simpler to not require the utils to be aware of the flask application here?

Well, is `util` supposed to be used outside of flask? I don't think so.

> is there a negaive to passing an additional parameter through here?

I thought it's not much use to add an argument that's identical on every call of the function.

> i tried and failed to use current_app, running into issues wiht the flask application context, so i also tried to fix that and failed.

You would need to create the application context in unit tests only, it's created for you when the webapp runs. ([explaination in the Flask docs](https://flask.palletsprojects.com/en/2.0.x/testing/#other-testing-tricks) )

> added it as an extra commit if you have suggestions, but TBH, would like to know if this is worth it to just avoid passing an extra parameter to the util function.

Yeah, it's not a problem, keep it if it simpler/cleaner for you.